### DISCUSSION TITLE
Webcam privacy glass stickers

### DISCUSSION THREAD

**Discussion by ryanprior** [elementary/store#153] [Product Ideas]
Glass stickers with the elementary logo can cover the webcam for extra privacy. Practical, cheap and on-message.

**Comment by matthiasjg** [comment]
I really think it would be nice to actually have the elementary logo in front of your OWN face w/ a branded webcam cover—instead of the usual sticker on the lid for the pleasure of the people around you. 

Of course, also protecting yourself from prying eyes!

Maybe even in black and white (think dark/ light mode) to match different laptop colors… :thinking:
